spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'json'
+require 'webmock/rspec'
+require 'support/vcr_setup'
+
+RSpec.describe BingApiV7 do
+  bing = BingApiV7.new
+  example = 'softserve'
+
+  describe '#search', vcr: true do
+    context 'with valid response data' do
+      let(:search_query) { example }
```

Comment:
let!

Reply:
no need to use bang here, as this is just a string value, not a factory